jobs/component_jobs.groovy

```diff
@@ -87,15 +100,19 @@ repos.each { Map repo ->
       }
 
       publishers {
-        def statuses = [['SUCCESS', 'pending'],['FAILURE', 'failure'],['ABORTED', 'error']]
-        postBuildScripts {
-          steps {
-            statuses.each { buildStatus, commitStatus ->
-              conditionalSteps {
-                condition {
-                  status(buildStatus, buildStatus)
-                  steps {
-                    shell curlStatus(buildStatus: buildStatus, commitStatus: commitStatus, jobName: name, repoName: repo.name)
+        if (isPR) {
+          def statuses = [['SUCCESS', 'pending'],['FAILURE', 'failure'],['ABORTED', 'error']]
+          postBuildScripts {
+            onlyIfBuildSucceeds(false)
+            steps {
+              statuses.each { buildStatus, commitStatus ->
+                conditionalSteps {
+                  condition {
+                    status(buildStatus, buildStatus)
+                    steps {
+                      shell StatusUpdater.curlStatus(
```

Comment:
I will say that I've verified `shell curlStatus(args…)` does work as intended (when `curlStatus` declaration is in same file) b/c that method just returns the string (comprising the shell script to execute) to the `shell` closure.  So, the only addition here is attempting to access `curlStatus` from a different file via the class it is declared in, `StatusUpdater`... but yes, still learning the nuances of Groovy